javascript/stimulus_reflex.js

```diff
@@ -304,7 +304,7 @@ const extendStimulusController = controller => {
 
 const registerReflex = data => {
   const { reflexId } = data
-  reflexes[reflexId] = { finalStage: 'finalize' }
+  reflexes[reflexId] = { finalStage: 'finalize', warned: false }
```

Comment:
Not sure we need to initialize `warned: false` but I really like what this PR accomplishes.

Reply:
When I did it, I think I was worried about cross-tab stuff... but you are correct, sir.